packages/react-components/react-accordion/src/components/Accordion/useAccordion.ts

```diff
@@ -10,18 +10,11 @@ import { useArrowNavigationGroup } from '@fluentui/react-tabster';
  * @param ref - reference to root HTMLElement of Accordion
  */
 export const useAccordion_unstable = (props: AccordionProps, ref: React.Ref<HTMLElement>): AccordionState => {
-  const {
-    openItems: controlledOpenItems,
-    defaultOpenItems,
-    multiple = false,
-    collapsible = false,
-    onToggle,
-    navigation,
-  } = props;
+  const { multiple = false, collapsible = false, onToggle, navigation } = props;
   const [openItems, setOpenItems] = useControllableState({
-    state: React.useMemo(() => normalizeValues(controlledOpenItems), [controlledOpenItems]),
-    defaultState: () => initializeUncontrolledOpenItems({ defaultOpenItems, multiple }),
-    initialState: [],
+    state: React.useMemo(() => normalizeOpenItems(props.openItems), [props.openItems]),
+    defaultState: undefined,
+    initialState: () => initializeUncontrolledOpenItems({ defaultOpenItems: props.defaultOpenItems, multiple }),
```

Comment:
why don't you use the user's default state as the default state?

Reply:
because the user's default state is not equivalent to the internally controlled state

Reply:
Sorry, but I still don't understand a benefit of doing that. Both `defaultState` and `initialState` are initialized lazily anyway. `defaultState` stays for user's input and there should be no difference in handling compared to `initialState`...

Reply:
`defaultState` is not initialized lazily, `defaultState` in this case is a well defined value. the function for lazy initialization is only available at `initialState`

Reply:
The way I see, the other alternative would be:

```ts
const [openItems, setOpenItems] = useControllableState({
  state: React.useMemo(() => normalizeOpenItems(props.openItems), [props.openItems]),
  // either this memoization or I'll invoke normalization on every render for  no reason
  defaultState: React.useMemo(
    () => initializeUncontrolledOpenItems({ defaultOpenItems: props.defaultOpenItems, multiple }),
    // eslint-disable-next-line react-hooks/exhaustive-deps
    [],
  ),
  initialState: [],
});
```

By simply using the lazy initialization provided by `initialState` we avoid the  unnecessary memoization (since that value is only consumed once), and that's why using `initialState` initialization over `defaultState` is more ideal in this particular cases where the value provided by the user is not equivalent to the internal state.

Reply:
Which is the same thing for the tree for example: 

https://github.com/microsoft/fluentui/blob/11eb9bab7a225d377a9c5f9448279ff44232a097/packages/react-components/react-tree/src/hooks/useOpenItemsState.ts#L7-L11

Reply:
<img width="691" alt="image" src="https://user-images.githubusercontent.com/14183168/236145671-cc87c717-cd4d-41d4-a945-252c92663ca8.png">

> `defaultState` is not initialized lazily, `defaultState` in this case is a well defined value. the function for lazy initialization is only available at `initialState`

Well, function was available before your changes. That's the point of confusion